.travis.yml

```diff
@@ -79,12 +79,6 @@ jobs:
         - SMOKETEST=1
         - RUN_SYNAPSE=1
 
-    - stage: test
-      <<: *job-template-test
-      env:
-        - BLOCKCHAIN_TYPE='tester'
```

Comment:
I would like to keep this test and just change to `geth` here.

Reply:
definitely this needs to be kept, it's the only stage that runs the unit tests

Reply:
ops good point. Wanted to only remove tester contracts integration there. Amending now.

Reply:
But I will remove the contracts tests. They run again at integration